### PR TITLE
(SIMP-3502) core: Fix Forge publishing

### PR DESCRIPTION
**Pull request by jeefberkey**
Previously, the travis.yml did not look like our other modules and did
not live in it's own.

The purpose of having simp-core have it's own puppet module is to behave
as a shortcut to install SIMP from the Puppet Forge. It only has
dependencies required to get started.

Full previous discussion can be found here:
https://github.com/simp/simp-core/pull/427

SIMP-3502 #close
SIMP-3881 #close

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -12,8 +12,10 @@
     "bundle",
     "security",
     "compliance",
-    "NIST 800-53",
-    "DISA STIG"
+    "NIST",
+    "800-53",
+    "DISA",
```
Comment (trevor-vaughan) [inline metadata.json]: These really shouldn't be split.

If spaces aren't allowed, use underscores.

Comment (jeefberkey) [reply metadata.json]: Why would anyone search for those? Splitting them up means they would show up in searches for any one of the tags, right?

Comment (trevor-vaughan) [reply metadata.json]: Hrm...yeah, I guess you're right. Just much less precise than would be nice. That said, OK!

**Comment by trevor-vaughan** [comment]
Rebased

**Pull request merged**